### DISCUSSION TITLE
Reference window

### DISCUSSION THREAD

**Discussion by Storm840** [keiwando/toonsquid#620] [Ideas]
Hi, It would be cool to be able to have a reference window where we could add images and drag around the canvas (kinda like in ibis paint). In my opinion, It would probably be better than to add the image directly in the canvas

**Comment by DWrightAnimation** [comment]
Sorry for not replying until now, but I think a reference window that can also do videos would be really helpful for those who have video reference, but either don't want to use multiple devices to view both the reference and their animation, or don't want to have a video layer take up file space and make it inconvenient for adding drawings one at a time.
